README.md

```diff
@@ -35,6 +35,11 @@ or `skopeo`:
   * `quay.io/containers/*:latest` and `quay.io/*/stable:latest` -
     Built daily using the same `Containerfile` as above.  The tool versions
     will remain the "latest" available in Fedora.
+  * `quay.io/containers/aio:latest` and `quay.io/containers/aio:<date stamp>` -
+    "All In One" image containing podman, buildah, and skopeo.  Built weekly
```

Comment:
I take my LGTM back!
```suggestion
    "All In One" image containing Podman, Buildah, and Skopeo.  Built weekly
```

Reply:
Oh lol, I was looking at that yesterday and thinking..."Should I capitalize them"?  :rofl: